src/configuration.rs

```diff
@@ -0,0 +1,13 @@
+// Configuration for generating evergreen tasks.
+
+#[derive(Debug, Clone)]
+pub struct GenerateConfig {
```

Comment:
I pulled the global settings into this struct instead of passing them around individually.